src/components/CovMap.tsx

```diff
@@ -168,14 +165,21 @@ export const CovMap = () => {
   const mappingLayers = Object.keys(visual.mappings);
   const handleMapClick = (pointerEvent, stateViewport) => {
     const { features } = pointerEvent;
+    console.log(features)
     if (features.length > 0) {
+      /* console.log(config.visuals.covmap.layers) */
+
+      /* handle multiple features. this happens when a street or text is clicked */
+      const countyFeatures = features.length > 1 ?
+        features.filter(feature => feature.layer && feature.layer.id == "areas-fill") : features;
```

Comment:
@kantimam 🎉Top! Koennen wir die layer id (hier `"areas-fill"`) noch einstellbar machen in `apps/official/index.ts`? Dann koennen wir einfacher reagieren wenn sich die Layer aendern.

Reply:
jo können wir war mir nicht sicher ob wir auch nach der zweiten "hover" layer suchen sollen aber glaube nicht oder?

Reply:
```
layers: [
        {
          id: "areas-fill",
          source: "CI-to-plz",
          // showLegend: true,
          fn: (dataField, timeKey) => ({
            type: LayerType.FILL,
            paint: {
              "fill-color": [
                "interpolate",
                ["linear"],
                ["get", dataField, ["get", timeKey]],
                1,
                "#219653",
                2,
                "#EEC341",
                3,
                "#E84C4C",
              ],
              "fill-opacity": 0.5,
            },
            beforeId: "road-label",
          }),
        },
        {
          id: "hover",
          source: "CI-to-plz",
          fn: () => ({
            type: LayerType.LINE,
            paint: {
              "line-color": "#627BC1",
              "line-width": ["case", ["boolean", ["feature-state", "hover"], false], 4, 0],
            },
          }),
        },
      ],
```

Reply:
Der hover Layer zeichnet die blaue Linie drum herum, soll ja nicht klickbar sein. Falls man aber etwas mit mehreren layern zeichnet, wäre es schon gut wenn man mehrere hinzufügen kann in der config.